app/src/main/res/layout/dialog_change_start_date.xml

```diff
@@ -3,6 +3,7 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tool="http://schemas.android.com/tools"
     android:layout_width="match_parent"
+    android:theme="@style/Hedvig.Theme"
```

Comment:
Is this the optimal way to apply the theme should be applied? Preferably, our theme would automatically be applied to all bottom sheets.

Reply:
dont know why i did that since that is what is does 🤷‍♀